metadata.json

```diff
@@ -40,7 +40,7 @@
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">= 4.6.1 <5.0.0"
+      "version_requirement": ">= 47.00 < 5.0.0"
```

Comment:
Are we managing this independently on each repo? Haven't seen it on the other modulesyncs